src/parse.rs

```diff
@@ -331,6 +331,17 @@ fn handle_unilateral<'a>(
         Response::Expunge(n) => {
             unsolicited.send(UnsolicitedResponse::Expunge(n)).unwrap();
         }
+        Response::Data {
+            status,
+            code,
+            information,
+        } if imap_proto::Status::Ok == status && code.is_none() => {
+            unsolicited
+                .send(UnsolicitedResponse::Uncategorised(
+                    information.unwrap_or("").to_string(),
```

Comment:
Huh, I'm surprised that the `information` is an `Option`. Judging from [the spec](https://tools.ietf.org/html/rfc3501#section-7.1.1) it doesn't seem like it can be left out? @djc